torch/testing/_internal/inductor_utils.py

```diff
@@ -39,9 +39,11 @@ def test_cpu():
 
 HAS_CPU = LazyVal(test_cpu)
 
-HAS_CUDA = torch.cuda.is_available() and has_triton()
+HAS_TRITON = has_triton()
```

Comment:
We don't need this, right?

Reply:
I just moved `requires_triton`, previously in `test/inductor/test_codecache.py`, into this file.